packages/js/client/src/wasm/WasmWeb3Api.ts

```diff
@@ -40,17 +41,21 @@ export class WasmWeb3Api extends Api {
     mutation?: ArrayBuffer;
   } = {};
 
+  private _sanitizedEnviroment?: ArrayBuffer;
```

Comment:
TBH it has been so long since I have done this that I don't remember the idea at all, but I guess sanitization is done from this branch: https://github.com/polywrap/monorepo/pull/576